runtime/calamari/src/lib.rs

```diff
@@ -998,14 +1000,101 @@ impl_runtime_apis! {
         fn dispatch_benchmark(
             config: frame_benchmarking::BenchmarkConfig
         ) -> Result<Vec<frame_benchmarking::BenchmarkBatch>, sp_runtime::RuntimeString> {
-            use frame_benchmarking::{Benchmarking, BenchmarkBatch, TrackedStorageKey};
+            use frame_benchmarking::{Benchmarking, BenchmarkBatch, TrackedStorageKey, BenchmarkError};
 
             use frame_system_benchmarking::Pallet as SystemBench;
             impl frame_system_benchmarking::Config for Runtime {}
 
             use cumulus_pallet_session_benchmarking::Pallet as SessionBench;
             impl cumulus_pallet_session_benchmarking::Config for Runtime {}
 
+            use pallet_xcm_benchmarks::asset_instance_from;
+            use xcm_config::{LocationToAccountId, XcmExecutorConfig};
+
+            parameter_types! {
+                pub const TrustedTeleporter: Option<(MultiLocation, MultiAsset)> = None;
+                pub const TrustedReserve: Option<(MultiLocation, MultiAsset)> = Some((
+                    KsmLocation::get(),
+                    MultiAsset { fun: Fungible(1_000_000_000_000), id: Concrete(KsmLocation::get()) },
```

Comment:
Can you leave a comment here, why `1_000_000_000_000` though you have explained it in the PR?

Reply:
Sure, this is just a random number, doesn't really matter